packages/xs-extra/xentoollog.master/opam

```diff
@@ -0,0 +1,18 @@
+opam-version: "2.0"
+synopsis: "Mock OCaml bindings for Xen's xentoollog"
+maintainer: "lindig@gmail.com"
+authors: "lindig@gmail.com"
```

Comment:
Is this correct? Are these the mock bindings? And maybe I should not be named as an author with my private address if this is in `xapi-project`.

Reply:
These are the mock bindings, I'll change the maintainer and authors